geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryStrategyDecider.scala

```diff
@@ -49,14 +52,24 @@ object QueryStrategyDecider extends QueryStrategyDecider with MethodProfiling {
    */
   override def chooseStrategies(sft: SimpleFeatureType,
                                 query: Query,
-                                hints: StrategyHints,
+                                stats: GeoMesaStats,
                                 requested: Option[StrategyType],
                                 output: ExplainerOutputType): Seq[Strategy] = {
 
     implicit val timings = new Timing()
 
     // get the various options that we could potentially use
-    val options = new QueryFilterSplitter(sft).getQueryOptions(query.getFilter, output)
+    val options = {
+      val all = new QueryFilterSplitter(sft).getQueryOptions(query.getFilter, output)
+      // don't evaluate z2 index if there is a z3 option, it will always be picked
```

Comment:
Ok, shout if I have this wrong:  spatial-only goes to Z2.  ST queries go to Z3.  

At some point, we could calculate costs and route long-temporal ST queries to Z2, but not now, right?